plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/security/realms/ldap/internal/persist/LdapConfigurationSource.java

```diff
@@ -12,15 +12,33 @@
  */
 package org.sonatype.security.realms.ldap.internal.persist;
 
-import java.io.IOException;
+import java.util.List;
 
-import com.sonatype.security.ldap.realms.persist.model.CLdapConfiguration;
-
-import org.sonatype.configuration.ConfigurationException;
+import org.sonatype.security.realms.ldap.internal.persist.entity.LdapConfiguration;
 
+/**
+ * Actual persistence mechanism of {@link LdapConfiguration}.
+ */
 public interface LdapConfigurationSource
 {
-  CLdapConfiguration load() throws ConfigurationException, IOException;
+  /**
+   * Loads all the configuration entries as a modifiable list with undefined order. Modifications to the list are not
```

Comment:
Removing it. It was more a note to myself, as some of the tests was intentionally modifying the list, while my mock implementation did return an immutable list.